lint/src/test/java/com/firebaseui/lint/NonGlobalIdDetectorTest.kt

```diff
@@ -1,14 +1,16 @@
 package com.firebaseui.lint
 
 import com.android.tools.lint.checks.infrastructure.TestFiles.xml
+import com.android.tools.lint.checks.infrastructure.TestLintTask
 import com.android.tools.lint.checks.infrastructure.TestLintTask.lint
 import com.firebaseui.lint.NonGlobalIdDetector.Companion.NON_GLOBAL_ID
 import org.junit.Test
+import java.io.File
 
 class NonGlobalIdDetectorTest {
     @Test
-    fun validId() {
-        lint()
+    fun `Passes on valid view id`() {
```

Comment:
My thoughts on this Kotlin language feature:
https://www.youtube.com/watch?v=9nazm3_OXac

(but seems idiomatic so fine to leave it)

Reply:
Haha, same. Yeah, this is taken straight from the Intellij [code style](https://kotlinlang.org/docs/reference/coding-conventions.html):
![image](https://user-images.githubusercontent.com/9490724/34497843-1ae0315a-efb4-11e7-9393-3c26605cf697.png)